hostprocess/csi-proxy/build.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+set -x
+set -e
+
+if [[ -z "${version}" ]]; then
+    echo "Required env var 'version' is not set"
+    exit 1
+fi
+echo "Using version ${version}"
+
+repository=${repository:-"ghcr.io/kubernetes-sigs/sig-windows"}
+
+docker buildx create --name img-builder --use --platform windows/amd64
+trap 'docker buildx rm img-builder' EXIT
+
+docker buildx build --platform windows/amd64 --output=type=registry -f Dockerfile.windows --build-arg=WINDOWS_VERSION=1809 -t ${repository}/csi-proxy:${version}-1809 .
```

Comment:
looks like we should be adding `--build-arg=CSI_PROXY_VERSION=$version` here to make sure we clone the correct csi proxy version

Reply:
I thought about this some... Should we have anything CSI_PROXY_VERSION file in the repo for this?
I can see a need for having the image version and csi-proxy version not stay in sync (adding new windows OS versions for example).

Reply:
We need a way to bump the csi version.  The way this is done in k/k test images is use the same version but append a `-1` to each change.  so for instance if we made a change the image but still using 1.0.1 version of kube-proxy the image tag would be `1.0.1-2` where `2` is the version we are building.  This isn't perfect but for our purposes should work?

Reply:
Right now I do have the csi-proxy version and the container image version decoupled.
The VERSION file is used for the container image version (ex `v1.0.2-2`) and the version of csi-proxy used is specified in the dockerfile.
Is this sufficient or do you think we should not have the version of csi-proxy used hardcoded in the Dockerfile?

Reply:
keeping it hard coded is probably fine we shouldn't support to many versions of the matrix